CRM/Contribute/Form/Contribution/Main.php

```diff
@@ -1310,20 +1310,30 @@ public function assignFormVariablesByContributionID() {
       return;
     }
     if (!$this->getContactID()) {
-      CRM_Core_Error::statusBounce(ts("Returning since there is no contact attached to this contribution id."));
+      CRM_Core_Error::fatal(ts("Returning since there is no contact attached to this contribution id."));
```

Comment:
We shouldn't be adding in fatals. It's a deprecated function.

Reply:
Modified it to use an exception instead of the above 👍